types/types.go

```diff
@@ -160,3 +160,36 @@ func (m ComMatrix) Contains(cd ComDetails) bool {
 
 	return false
 }
+
+// Returns True if both ComMatrixes have exactly the same ComMatrix details.
+func (m ComMatrix) Equals(other ComMatrix) bool {
```

Comment:
why not to use the Diff function?
u need from diff to be equal to nothing from 2 sides

Reply:
@sabinaaledort also can use that on the test of the origin

Reply:
Because we don't need the Diff matrices is this case, it occupies unnecessary space.